### PR TITLE
Redirect museum users to collections page after logging in and ordering

### PR DESCRIPTION
**Pull request by aahonen19**
- Redirect museum users to public collections page after logging in.
- Redirect museum users to public collections page after they have
  made successful order

### CODE REVIEW

**Comment by aahonen19** [comment]
Maybe we should display some message to museum user after they have placed an order? Or display the `checkout_complete` PageContent page and redirect to collections page after 60s instead of front page?

**Comment by aslani** [comment]
Yeah, you are right. So basically I guess the actual need is when ever the museum/kiosk user is going to / or frontpage view the user needs to be redirected to albums. So yes, actual checkout complete page should be displayed as normal

File: hkm/views/views.py [hunk 1]
```diff
@@ -1367,8 +1373,7 @@ def handle_checkout(self, request):
             self.send_notification_email(order_collection)
 
             self.request.basket.clear_all()
-            page_content = PageContent.objects.get(identifier="checkout_complete")
-            return self.render_to_response(self.get_context_data(form=form, page_content=page_content, order=order_collection))
```
Comment (aslani) [inline hkm/views/views.py]: So yeah, I think just leave this actually here

File: hkm/views/views.py [hunk 2]
```diff
@@ -97,7 +97,13 @@ def post(self, request, *args, **kwargs):
     def handle_login(self, request, *args, **kwargs):
         form = django_forms.AuthenticationForm(request, data=request.POST)
         if form.is_valid():
-            auth_login(request, form.get_user())
+            user = form.get_user()
+            auth_login(request, user)
+
+            # Redirect museum users to collections page after logging in
+            if user.profile.is_museum:
+                return redirect("hkm_public_collections")
```
Comment (aslani) [inline hkm/views/views.py:105]: This 👍

**Comment by aahonen19** [comment]
museum/kiosk user is now only redirected to collections page after logging in. If needed we can do middleware which checks if user is museum/kiosk user and it trying to go the front page, then we can redirect them to the collections page so they cannot ever go the front page.

**Comment by aslani** [comment]
I think this is fine for now

**Pull request merged**